.github/workflows/ci.yaml

```
@@ -28,7 +28,7 @@ jobs:
           fetch-depth: 0
       - name: Set up Python ${{ env.DEFAULT_PYTHON }}
         id: python
-        uses: actions/setup-python@v2.3.2
+        uses: actions/setup-python@v3
```

```suggestion
        uses: actions/setup-python@v3.0.0
```
We should pin `3.0.0` everywhere instead to prevent accidental regressions.

---

Done!